### ISSUE TITLE
Docs do not use the evergreen Discord invite link to the Mina Protocol server [Welcome & MINA sections] 

### ISSUE THREAD

**Issue by barriebyron** [o1-labs/docs2#293]
Docs do not consistently use the standard evergreen Discord invitation link to the Mina Protocol server. 

Now that the Mina Discord server has an [evergreen invite](https://discord.gg/minaprotocol) link `https://discord.gg/minaprotocol` it's time to use it everywhere. We can build trust and reliability by removing the possibly expired and inconsistent invite links that are scattered throughout the docs.

This issue applies only to these docs sections:

- [Welcome](https://docs.minaprotocol.com/zkapps/tutorials/hello-world#) 
- [About MINA](https://docs.minaprotocol.com/zkapps/tutorials/hello-world#)
- [Using MINA](https://docs.minaprotocol.com/zkapps/tutorials/hello-world#) 

Companion issue #294 

Excluded are the Node Operator-related sections, as these sections are undergoing improvement separately. We will also want to address tense in the content there eventually but now is not the time.

**Comment by nicc** [comment]
Indirectly Irresponsible Individual: @jasongitmail

**Comment by barriebyron** [comment]
Most existing channel links are not links to the channel itself but links to the Mina Protocol Discord server. 

Using descriptive link text guidelines, let's instead move the descriptive text link to the Discord server. 

I've landed on using this syntax for referring to channel names when the channel name is mentioned in the text that links out to the Discord invite. 

Key point: I omitted the article "the" and omitted "server"

> ...join the **#ledger-hardware** channel on [Mina Protocol Discord](https://discord.gg/minaprotocol).
